### ISSUE TITLE
some problem of test and train data

### ISSUE THREAD

**Issue by edision96** [khundman/telemanom#15]
I can not find test and train data from these files. My graduation project will use your model, but I can't find your training set and test set. I want to understand the input format of the entire code through this.
I look forward to hearing back.

**Comment by vc1492a** [comment]
@edision96 the repository readme details how to retrieve the data, immediately following the command on how to clone the repository. Simply run:

```shell
curl -O https://s3-us-west-2.amazonaws.com/telemanom/data.zip && unzip data.zip && rm data.zip
```
The data will be in the `train` and `test` directories once you cd into `data`.

**Comment by edision96** [comment]
Thank you very much for your help. I have now found the data for the training set and the test set. Thanks again for your help.

**Comment by edision96** [comment]
Are the data in the training set and the test set the same? Why does my code test my data not quite right？

**Comment by edision96** [comment]
@vc1492a Are the data in the training set and the test set the same? Why does my code test my data not quite right？

**Comment by khundman** [comment]
@edision96, the train and test data are the same shape but are different lengths and have different values. We can't help you if we don't know anything about the errors or issues you are encountering. Please provide details on your setup, errors, etc.

**Comment by khundman** [comment]
Closing due to inactivity.
